DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: etn
 Title: Access Data from the European Tracking Network
-Version: 2.2.1
+Version: 2.2.2
```

Comment:
If you bump here, don't forget to bump in CITATION.cff before you release. Same with DOI in CITATION.cff, which is tricky, since you don't know it yet. I have no solution for this.

Reply:
I used the general, all version DOI in my citation.cff. Was that a mistake?

Reply:
Ah, didn't catch that. That works! 👍